src/ensembl/src/styles/_settings.scss

```diff
@@ -39,6 +39,7 @@ $mustard: #cc9933;
 $green: #47d147;
 
 $global-box-shadow: $medium-light-grey;
+$form-field-shadow: inset 2px 2px 4px -2px $medium-dark-grey, inset -2px -2px 1px -2px $dark-grey;
```

Comment:
Should it be `inset 2px 2px 4px -2px $medium-dark-grey, inset -2px -3px 3px -2px $light-grey;` to match with the XD?

Reply:
Andrea is happy with the current styling.

Reply:
Checked during code review w/Andrea, and decided to leave the current value.